packages/office-ui-fabric-react/src/components/GroupedList/GroupHeader.base.tsx

```diff
@@ -18,7 +18,7 @@ export interface IGroupHeaderState {
 
 export class GroupHeaderBase extends BaseComponent<IGroupHeaderProps, IGroupHeaderState> {
   public static defaultProps: IGroupDividerProps = {
-    expandButtonProps: { 'aria-label': 'expand collapse group' }
+    expandCollapseButtonAriaLabel: 'expand collapse group'
```

Comment:
I was concerned about the string over here. I confirmed with David that at least in his mental model, Fabric avoids having non-localized strings baked into the component even as defaults. We instead provide these in all the component examples... 

Tag @dzearing to see if he has anything more to add.

Reply:
Yeah, I agree. I don't think we should bake nonlocalized strings into components

Reply:
In any case, that is a bad string... we should use the positive action of the button for the `aria-label` and use metadata to deliver the on/off state.

For `role=button`, `aria-pressed=[state]`

For `role=toggle` or `role=checkbox`, `aria-checked=[state]`

Reply:
Do you think if we use the `aria-expanded` on this expand/collapse button (as mentioned in my comment on `GroupedListSection`), that would be sufficient? Instead of `aria-pressed`? Or we could have both, but I'm not sure if that would be confusing.

Reply:
@natalieethell See https://github.com/OfficeDev/office-ui-fabric-react/pull/5794#discussion_r208011786 above comment for nuance around "the row" and "the twisty on the row"